README.md

```diff
@@ -544,6 +545,26 @@ password  | String | Cloudant password
 url       | String | Cloudant URL containing both username and password
 modelIndex | String | Specify the model name to document mapping, defaults to `loopback__model__name`
 
+### iamApiKey
+
+We honor the plugins’ configuration and passes them to the driver. 
+[Cloudant plugins](https://github.com/cloudant/nodejs-cloudant#the-plugins) has section 'iamauth' to show how the iam authentication is handled by the driver.
+
+To enable the iam connection, you can configure your LoopBack datasource as:
```

Comment:
Since we're not enabling the IAM, would it be better with the following:
```
To connect to the Cloudant database using IAM authentication, you can configure your LoopBack datasource as:
```